### DISCUSSION TITLE
Use case for Cheroot and documentation

### DISCUSSION THREAD

**Discussion by jkoppolu** [cherrypy/cheroot#644] [Q&A] [answered]
Hi Everyone, 
Noob question. Is this project usable as a stand-alone application server for a Django project? 
Or this project's scope is limited to use for Cherrypy only?

Could not find any normie `how to` documentation.

**Comment by webknjaz** [comment, marked as answer]
Cheroot can serve any WSGI app. There is an example in the CherryPy docs @ https://docs.cherrypy.dev/en/latest/advanced.html#host-a-foreign-wsgi-application-in-cherrypy. Though, that one does not showcase using just Cheroot.
There are Cheroot + Flask examples on SO: https://stackoverflow.com/a/50949979/595220 / https://stackoverflow.com/a/55367706/595220.
Essentially, you need to import your WSGI app and wire it into the server/routing. The framework wouldn't matter for as long as it implements the WSGI interface. So Django would work just as well.

There's also an executable that is called `cheroot` that you can use — some examples are at the beginning of https://cheroot.cherrypy.dev/en/latest/pkg/cheroot.cli/.
